Trading/Exchange/binance/binance_exchange.py

```diff
@@ -276,18 +276,6 @@ def fix_trades(self, raw, **kwargs):
     def parse_position(self, fixed, force_empty=False, **kwargs):
         try:
             parsed = super().parse_position(fixed, force_empty=force_empty, **kwargs)
-            parsed[trading_enums.ExchangeConstantsPositionColumns.MARGIN_TYPE.value] = \
-                trading_enums.MarginType(
-                    fixed.get(ccxt_enums.ExchangePositionCCXTColumns.MARGIN_MODE.value)
-                )
-            # use one way by default.
-            if parsed[trading_enums.ExchangeConstantsPositionColumns.POSITION_MODE.value] is None:
-                parsed[trading_enums.ExchangeConstantsPositionColumns.POSITION_MODE.value] = (
-                    trading_enums.PositionMode.HEDGE if fixed.get(ccxt_enums.ExchangePositionCCXTColumns.HEDGED.value,
-                                                                  True)
-                    else trading_enums.PositionMode.ONE_WAY
-                )
-            return parsed
```

Comment:
It's not necessary anymore right?

Reply:
right, it's there now https://github.com/Drakkar-Software/OctoBot-Trading/pull/1126/files#diff-e5b326dc37c0f13e228463fcd9ab3d5bcf3468d73b023398dfdc2130eaa671bdR281